### DISCUSSION TITLE
【多个数据库】能不能再链接第二个notion page id？比如一个notion图片数据库？一个博客内容，一个主页图片内容

### DISCUSSION THREAD

**Discussion by goldeye0351** [tangly1024/NotionNext#2054] [Ideas]
**为什么提出这个新的特性改动**
我想实现自动引用notion图片数据库中的图片和href来更新我的主页图片和相应链接。我的主页：http://www.535251.xyz 

**描述一下你推荐的解决方案**
不知道,我现在是直接引用的图片绝对地址

**描述一下你考虑过的其它替代解决方案**
不知道,我现在是直接引用的图片绝对地址

**补充说明**
我现在如果要修改图只能修改绝对地址，我想直接引用notion中的图

**Comment by miaoyc666** [comment]
我也非常想要这个特性！
